### PR TITLE
Update edit_distance.cpp to take input from user

### PR DESCRIPTION
**Pull request by yash-shimpi**
#### Description of Change
Fix: Now takes string input from user rather than predefined strings

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Review by Panquesito7 — COMMENTED** [review]
Thanks for contributing! 🚀
Would you like to take the opportunity to update this code to match our [**repository standards**](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md) with proper documentation and refined code?

Let us know if you need any help here or on our [**Discord server**](https://the-algorithms.com/discord/)! 😄

**Comment by yash-shimpi** [comment]
> Thanks for contributing! 🚀 Would you like to take the opportunity to update this code to match our [**repository standards**](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md) with proper documentation and refined code?
> 
> Let us know if you need any help here or on our [**Discord server**](https://the-algorithms.com/discord/)! 😄

Ok sure

**Comment by Avimathu** [comment]
Hi can I take up this task?

**Comment by Panquesito7** [comment]
> Hi can I take up this task?

Only if @yash-shimpi is not going to work on it further.

File: dynamic_programming/edit_distance.cpp [hunk 1]
```diff
@@ -75,8 +75,10 @@ int editDistDP(string str1, string str2, int m, int n) {
 }
 
 int main() {
-    string str1 = "sunday";
-    string str2 = "saturday";
+    string str1;
+    string str2;
+
+    cin >> str1 >> str2;
```
Comment (krishnacx) [inline dynamic_programming/edit_distance.cpp:81]: Can we have interactive string inputs:

`
cout << "Enter first string: ";
cin >> str1 ;

cout << "Enter second string: ";
cin >> str2 ;`

**Pull request closed without merging**